src/app/components/shared/search-project/search-project.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="form-group">
+    <input (keyup)="changeFilterValue()" type="text" class="form-control" placeholder="&#xF002; Search project" name="filterProject" [(ngModel)]="filterProject" style="font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif, FontAwesome">
```

Comment:
Please, fix the indentation, remember that we are using an indentation based in 2 spaces.

Reply:
done